### DISCUSSION TITLE
Consider releasing 1.0.0 version for next "major" release

### DISCUSSION THREAD

**Discussion by thenonameguy** [F1bonacc1/process-compose#82] [General]
Since the project is getting [widespread](https://github.com/Platonic-Systems/process-compose-flake) [adoption](https://www.jetpack.io/devbox/docs/cli_reference/devbox_services_start/) now and is quite stable in its core feature-set IMO it's time for it :)

Now the controversial part:
In the Clojure community (which is my main language of choice) semantic versioning is now generally eschewed.
If you want to read about some whys, here is a [short version of the argument](https://github.com/ptaoussanis/encore/blob/master/BREAK-VERSIONING.md) and [a long video of it, about general software compatibility](https://www.youtube.com/watch?v=oyLBGkS5ICk).

Just food for thought for future version numbers 🙏

**Comment by F1bonacc1** [comment]
Yes, I think that once I put all the documentation in order, instead of the huge mess that README.md became, PC can move to 1.0.0.
